Code/Framework/AzQtComponents/AzQtComponents/Buses/DragAndDrop.h

```diff
@@ -60,43 +74,84 @@ namespace AzQtComponents
         static const AZ::EBusAddressPolicy AddressPolicy = AZ::EBusAddressPolicy::ById;
         ///////////////////////////////////////////////////////////////////////
 
-        virtual int GetPriority() const
+        virtual int GetDragAndDropEventsPriority() const
         {
-            // our default handlers will return 0 for this.  You can override default behavior
-            // by making sure your handler gets invoked first (higher priority).
+            // default fall back 'last ditch effort' handlers will return priority 0.
+            // In general, if you provide specific override behavior and want to suppress the default fallback
+            // behavior, connect a listener with priority > 0.
             return 0;
         }
 
         ////////////////////////////////////////////////////////////////////////////////////////////
         //! Compare function required by BusHandlerOrderCompare = BusHandlerCompareDefault
         inline bool Compare(const DragAndDropEvents* other) const
         {
-            return GetPriority() > other->GetPriority();
```

Comment:
lawsonamzn note:  You tend to multi-inherit from many busses.  As such "GetPriority" is not a good name.  It needs to be unique amongst all the different busses you may multi-inherit from